permission_validators/asset/burn/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "iroha_asset_burn_validator"
+version = "2.0.0-pre-rc.13"
+authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
+edition = "2021"
```

Comment:
While you're at it, I think these can be [inherited](https://github.com/hyperledger/iroha/issues/3232) from workspace

Reply:
Cool feature